xstatic_bower.py

```diff
@@ -150,7 +150,7 @@ def main():
     underline = '-' * len('xstatic-' + display_name)
     homepage = bower_json['homepage']
     if 'main' in bower_json:
-        main = "MAIN='{}'".format(bower_json['main'])
+        main = "MAIN=\"{}\"".format(bower_json['main'])
```

Comment:
I think we need to format the repr() of the main var from the bower_json so strings and lists will both be correctly represented. Lose the quotes forced into the resultant value.